Automattic/jetpack
# IDC: Complete notice styles

Closes #5494

This PR should finalize the design for the notice. Here are screenshots:

### Step one
<img width="739" alt="screen shot 2016-11-03 at 3 24 27 pm" src="https://cloud.githubusercontent.com/assets/1126811/19983873/c8e2cec2-a1d9-11e6-92eb-46528961ee5e.png">

<img width="1054" alt="screen shot 2016-11-03 at 3 24 47 pm" src="https://cloud.githubusercontent.com/assets/1126811/19983893/de7544d6-a1d9-11e6-8f00-825fda391470.png">

### Step two

<img width="739" alt="screen shot 2016-11-03 at 3 24 37 pm" src="https://cloud.githubusercontent.com/assets/1126811/19983881/cd4398ac-a1d9-11e6-99fa-693436008f3e.png">

<img width="1058" alt="screen shot 2016-11-03 at 3 24 53 pm" src="https://cloud.githubusercontent.com/assets/1126811/19983900/e3fac732-a1d9-11e6-8d1d-2725ad4daca8.png">


cc @jeffgolenski and @rickybanister for design review.

To test:

- Checkout this branch
- Put your site into an IDC (We have a test plugin to help with this if you're having an issue. Please ping)
- Click "Fix Jetpack's connection" and land on second step
- Reload page
- Click dismiss and verify notice is dismissed
